### PR TITLE
Nebula Dev Merge

### PR DESCRIPTION
**Pull request by NataKilar**
Merges us onto latest Nebula dev. Fission is in - Praise be!

### CODE REVIEW

File: mods/persistence/modules/mob/new_player.dm [hunk 1]
```diff
@@ -172,14 +172,6 @@
 	var/datum/skillset/SS = new_character.skillset 	// Populate the skill_list of the player's skillset so that they can be properly adjusted during gameplay.
 	SS.set_skillset_min()
 
-	new_character.dna.ready_dna(new_character)
```
Comment (PsyCommando) [inline mods/persistence/modules/mob/new_player.dm:175]: Why remove the dna setup there? It syncs dna with organs and all.

**Review by PsyCommando — CHANGES_REQUESTED** [review]
Looks alright. Just not sure why you removed the dna setup? Can you take a look?

**Review by PsyCommando — APPROVED** [review]
Alright looks good to me now!

**Review by PsyCommando — APPROVED** [review]

**Review by PipeWizard — APPROVED** [review]
Obligatory approval comment (thanks for the review psy)

**Pull request merged**